src/components/LoginRegisterForm/RegisterForm.js

```diff
@@ -0,0 +1,86 @@
+import { useState, useContext } from "react";
+import { Link } from "react-router-dom";
+import ApiContext from "../../contexts/ApiContext";
+import "./login-register-form.scss";
+
+const RegisterForm = () => {
+  const api = useContext(ApiContext);
+  const [msg, setMsg] = useState(null);
+
+  const handleSubmit = async (event) => {
+    event.preventDefault();
+    const { target } = event;
+    const [login, password, re_password] = target;
```

Comment:
if you started working with the form, then work to the end, otherwise it looks ridiculous.if you have only 3 fields, then why do you need a form at all, just work with the object in the state